internal/policy/container/has_modified_files_test.go

```diff
@@ -402,6 +405,37 @@ var _ = Describe("HasModifiedFiles", func() {
 		})
 	})
 
+	When("multiple no-op layers with the same IDs split layers containing RPMDB modifications", func() {
```

Comment:
Non-blocking: would this be better as a CI test?

Reply:
I think that might be more complex then it's worth after looking at it more.

Reply:
Our CI tests typically involve a cluster, no? We don’t need a cluster here.

Reply:
Our CI also builds images, which I forgot was 'our' image, ie preflight. I thought it could be any image/test combo, which only runs in prow and does not invoke a cluster.